soc/nordic/nrf53/Kconfig

```diff
@@ -163,10 +163,11 @@ config NRF_SPU_RAM_REGION_SIZE
 	  RAM region size for the NRF_SPU peripheral
 
 config SOC_NRF_GPIO_FORWARDER_FOR_NRF5340
-	bool
+	bool "Forward GPIO pins to network core"
```

Comment:
```suggestion
	bool "Forward GPIO pins control from network core"
```

Reply:
`to` is correct here, without this Kconfig, they remain under control of the application core, the network core needs the ability to control them to it is to the network core, not from the network core